app/models/admin.rb

```diff
@@ -26,10 +27,10 @@ class Admin
   text_search_in :name, :email
 
   validates_uniqueness_of :email, case_sensitive: false
-  validates_presence_of :encrypted_password
+  validates_presence_of :email
```

Comment:
Rails/Validation: Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.